src/screens/FeedScreen/Content.js

```diff
@@ -53,6 +53,7 @@ const Content = ({
   const [amountCut, setAmountCut] = React.useState(0);
   const [textCut, setTextCut] = React.useState(null);
   const [arrText] = React.useState([]);
+  const layoutHeight = dimen.size.FEED_CONTENT_HEIGHT;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The constant `layoutHeight` has been moved from state to a constant value. This change could potentially affect the component's re-rendering behavior. Make sure this does not negatively impact the component's functionality.
```diff
+ const layoutHeight = dimen.size.FEED_CONTENT_HEIGHT;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->